usaspending_api/search/v2/views/spending_by_award_count.py

```diff
@@ -150,3 +151,26 @@ def query_elasticsearch_for_prime_awards(self, filters) -> list:
             "other": other,
         }
         return response
+
+    def query_elasticsearch_for_subawards(self, filters) -> list:
+        query_with_filters = QueryWithFilters(QueryType.SUBAWARDS)
+        filter_query = query_with_filters.generate_elasticsearch_query(filters)
+        s = SubawardSearch().filter(filter_query)
```

Comment:
Not required, but you can set `size` to 0 for this query, since we only care about the aggregation buckets and not the individual hits.

```suggestion
        s = SubawardSearch().filter(filter_query).extra(size=0)
```